ui/ducks/confirm-transaction/confirm-transaction.duck.js

```diff
@@ -118,8 +130,32 @@ export default function reducer(state = initState, action = {}) {
         ...state,
         nonce: action.payload,
       };
-    case CLEAR_CONFIRM_TRANSACTION:
-      return initState;
+    case SET_DETAILS_FOR_CONFIRM_TX: {
+      return {
+        ...state,
+        sendTxDetailPerId: {
```

Comment:
Rather than coupling this duck to the send one, should we instead treat this as a simple confirm transaction flag per transaction.

So maybe `SET_MAX_VALUE_MODE` which is an object of transaction ID to boolean?

Reply:
Done in https://github.com/MetaMask/metamask-extension/commit/f362be605aa5deda59e74b27bfe810a05b60db8e